docs/admin/disruptions.md

```diff
@@ -61,17 +61,17 @@ itself.  To attempt an eviction (perhaps more REST-precisely, to attempt to
 {
   "apiVersion": "policy/v1alpha1",
   "kind": "Eviction",
-  "name": "quux",
-  "namespace": "default"
+  "metadata": {
+    "name": "quux",
+    "namespace": "default"
+  }
 }
 ```
 
 and here is how you would attempt this with `curl`
```

Comment:
Since at this point there's no other method available (in 1.5 there will be a go-client) I'll drop the also from that sentence.